<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="0">
    <widget class="QSpinBox" name="maxConsecutiveRayLength"/>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>maxRayLength</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QSpinBox" name="minRayLength">
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="recursionDepth">
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="debugDrawGaps">
     <property name="text">
      <string>debug draw gaps</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QSpinBox" name="maxRayLength">
     <property name="maximum">
      <number>5000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="debugDrawPolygon">
     <property name="text">
      <string>debug draw polygon</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="debugDrawRecursiveStartPoints">
     <property name="text">
      <string>debug draw recursive start points</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>recursionDepth</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="debugDrawRays">
     <property name="text">
      <string>debug draw rays</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QSpinBox" name="minRecursionRayLength">
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>maxConsecutiveRayLength</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="debugDrawFloodFillPoints">
     <property name="text">
      <string>debug draw flood fill points</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>stepSize</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>minRayLength</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QSpinBox" name="minRayDifferenceForRecursion"/>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>minRayDifferenceForRecursion</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>minRecursionRayLength</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDoubleSpinBox" name="stepSize">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>80.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
